creator-node/src/components/tracks/tracksComponentService.js

```diff
@@ -1,14 +1,16 @@
 const path = require('path')
 
+const config = require('../../config.js')
 const { logger: genericLogger } = require('../../logging')
 const { getSegmentsDuration } = require('../../segmentDuration')
 const TranscodingQueue = require('../../TranscodingQueue')
 const models = require('../../models')
 const DBManager = require('../../dbManager')
-const { removeTrackFolder, saveFileToIPFSFromFS } = require('../../fileManager')
+const fileManager = require('../../fileManager')
```

Comment:
for sinon to work in our test suite, we import the entire export VS the destructured export